web_switch_company_warning/static/src/js/switch_company_warning.js

```diff
@@ -0,0 +1,54 @@
+odoo.define('web_switch_company_warning.widget', function (require) {
+    'use strict';
+
+    var Widget = require('web.Widget');
+    var UserMenu = require('web.UserMenu');
+    var session = require('web.session');
+    //Show a big banner in the top of the page if the company has been
```

Comment:
[W7903(javascript-lint), ] Expected space or tab after '//' in comment. [Error/spaced-comment]